\documentclass{article}
\usepackage{amsmath,amsfonts,amsthm}
\usepackage{graphicx}
\usepackage{hyperref}
\usepackage{setspace}
\usepackage{caption}
\usepackage{subcaption}
\graphicspath{ {../results/} }
\usepackage[margin=1in]{geometry}

\onehalfspacing
\begin{document}
\author{Frank Ockerman}
\title{BIOS 611 Project}
\maketitle
\tableofcontents

\section*{Abstract}
Chronic obstructive pulmonary disease (COPD) has a high public health burden and is a leading cause of death worldwide. Although COPD is characterized by its pulmonary symptoms, previous studies have identified significant metabolomic dysregulations in COPD patients. In this project, I used metabolomic and phenotypic data from the Plasma Metabolomic Signatures of COPD Study to investigate metabolites and metabolic pathways associated with COPD status. A PLS-DA model replicated several literature metabolite-COPD associations and suggested novel avenues for further investigation. Application of GSEA demonstrated enrichment of COPD associations among sex steroid hormones and related metabolites.

\addcontentsline{toc}{section}{Abstract}

\section*{Introduction}
Common symptoms of chronic obstructive pulmonary disease (COPD) include wheezing, coughing, and difficulty breathing. But there are less obvious, systemic effects of COPD, notably including metabolomic dysfunction. Previous studies have implicated a host of metabolomites, including ceramides, branch chain amino acids, the TCA cycle, lipoproteins, and sex-specific hormones \cite{godboleMetabolomeFeaturesCOPD2022}\cite{tamRoleFemaleHormones2011}. Despite these results, many COPD-associated metabolites fail to replicate across studies, and the clinical significance of robustly-associated metabolites is typically unclear.

In this project, I attempted to identify metabolites associated with COPD status using data from the The Subpopulations and Intermediate Outcome Measures in COPD Study (SPIROMICS) \cite{gillenwaterPlasmaMetabolomicSignatures2020}. The metabolomic data in this study was generated by Metabolon using their proprietary system for non-targeted LC-MS profiling, which is intended to represent a broad spectrum of a patient's blood metabolites. Metabolomic data included a variety of annotations, including sub-pathways, super-pathways, and a variety of identifiers for linking metabolites to external databases. Study participants were non-Hispanic white and African American individuals aged between 40-80 years with at least 20 pack-years of smoking. 202 additional participants had no history of smoking. 649 subjects who returned for visit 5 of the study and consented to a fasting blood draw for metabolomic analysis were included in this study. Phenotypes for this data including self-reported race, gender, BMI, and smoker status (previous vs. current). I acquired this data using the Metabolomics Workbench's National Metabolomics Data Repository. This data is available at the NIH Common Fund's National Metabolomics Data Repository (NMDR) website, the Metabolomics Workbench, https://www.metabolomicsworkbench.org where it has been assigned Project ID PR001048. The data can be accessed directly via it's Project DOI: \href{http://dx.doi.org/10.21228/M87D6G}{10.21228/M87D6G}. That work is supported by NIH grant U2C-DK119886.

\addcontentsline{toc}{section}{Introduction}

\section*{Methods}
\addcontentsline{toc}{section}{Methods}
After combining data across assays and removing metabolites with greater than 50\% missingness, I imputed missing values using the minimum observed value for each metabolite. This choice is typical for analyses of non-targeted LC-MS metabolomic data and reflects the fact that most missing data is occurs when metabolite values are below the lower limit of detection. I then performed \(\log_{10}\) transformation and standardization on each metabolite. Upon visual inspection, this produced distributions with approximately Gaussian distributions for most metabolites. I obtained the principal component projection of the metabolomic data and used this to calculate the Mahalanobis distance for each subject. Briefly, Mahalanobis distance extends the concept of a standard deviation to summarize how far a subject is from the center of a multivariate Gaussian distribution. Based on the boxplot of these Mahalanobis distances (Figure \ref{fig:mahDist}), I identified a clear outlier, which I removed from subsequent analyses. After removing this subject, I repeated the transformation and standardization step on the original data.

To understand basic patterns of metabolomic variation, I performed two analyses: a heatmap of metabolite correlations, labelled by super-pathways, and a plot of metabolomic principal components, with individuals labeled by their COPD status. Correlation heatmaps can reveal blocks of related variables and characterize the extent of multi-collinearity in the data. The principal components plot may be used to identify any obvious clusters of participants, especially if they cluster by COPD status.

One of my priorities was to investigate metabolomic sub-pathways which are dysregulated in participants with COPD. I considered this analagous to the common problem in genetics of identifying gene sets which are differentially expressed according to some disease state. A typical approach to the gene set problem is Gene Set Enrichment Analysis (GSEA) \cite{subramanianGeneSetEnrichment2005}. Briefly, genes (in our case metabolites) are ranked by some statistic. A typical choice is log 2 fold change, which I use as well. GSEA calculates an enrichment score for each pathway that represents the degree to which genes (metabolites) in that pathway are over-represented at the beginning and end of the list. Pathways with a significant score are considered enriched for the disease or trait of interest. I implement this algorithm with the clusterProfiler R package \cite{wuClusterProfilerUniversalEnrichment2021}.

My other priorities were to model COPD status using metabolomic data and to identify individual metabolites associated with COPD status. To address these aims, I fit a sparse partial least squares discriminant analysis (SPLS-DA) model. Partial least squares (PLS) is a common technique which represents the independent and dependent variables in a new space such that the independent variables explain the maximum variance of the dependent variables. In this way, PLS can perform both dimensionality reduction and prediction, although it is highly prone to overfitting. PLS-DA \cite{perez-encisoPredictionClinicalOutcome2003} is an analagous method in the case where the dependent variables are categorical (as is the case for COPD status). To improve parsimony in the model, PLS-DA can be extended to sparse PLS-DA, or SPLS-DA. I implemented this method using the mixOmics R package \cite{rohartMixOmicsPackageOmics2017a}.

\begin{figure}
  \centering
  \caption{Boxplot of Mahalanobis Distance}
  \includegraphics[width=12cm]{outliers.png}
  \label{fig:mahDist}
  \end{figure}

\section*{Results}
\addcontentsline{toc}{section}{Results}

A descriptive summary of the phenotypic data is provided in Table \ref{tab:phensum}. I observed a case-control imbalance, with 374 controls to 262 cases. A minority (33.5\%) of subjects were current smokers, and the remainder were classified as former smokers. The majority of subjects (75.9\%) self-identified as White, a substantial minority (19\%) identifed as African American, and a small percentage of participants were recorded as American Indian, Alaskan Native, Asian, or Mixed race. Nearly 75\% of subjects had BMI \(>\) 25.0, a crude indicator that they could be overweight. A small majority (53.9\%) of participants are male.

A boxplot summarizing the number of metabolites in each super-pathway, colored by assay type, is provided in Figure \ref{fig:metabSum}. A plurality (N=231) of metabolites are involved in lipid metabolism. The other large super-pathway is amino acid metabolism (N=209). 130 metabolites are xenobiotics, a broad class which encompasses foreign chemicals that are not naturally produced by the body, such as pharmaceuticals. Xenobiotics are notable for having substantially greater missingness than other super-pathways \ref{fig:pathMiss}. Among xenobiotics with less than 50\% missingness, I observed a median missingness of 13.42\%. The super-pathway with the next highest median missingness was peptides (8.1\%).

\input{../results/phenoSummary.tex}

\begin{figure}
  \centering
  \caption{Metabolites Summary}
  \includegraphics[width=0.9\textwidth]{metaboliteCount.png}
  \label{fig:metabSum}
  \end{figure}

\begin{figure}
  \centering
  \caption{\% Missing By Superpathway}
  \includegraphics[width=\textwidth]{pathSummary.png}
  \label{fig:pathMiss}
  \end{figure}

The heatmap of metabolite correlations is provided in Figure \ref{fig:heat}. Two clusters of highly-correlated lipid-related metabolites are obvious, corresponding to blocks at the upper left and bottom right corners of the map. Another block of highly-correlated xenobiotics is observed towards the center of the map. The structured, pathway-specific patterns of highly-correlated metabolites indicate that multicollinearity is an important concern for multivariate modelling approaches. The plot of the principal components of metabolomic variation \ref{fig:pca} does not reveal any obvious clusters among the participants, and I do not identify any trend in the principal components by COPD status.

\begin{figure}
  \centering
  \begin{subfigure}[b]{0.45\textwidth}
    \caption{Metabolite Correlation Heatmap}
    \includegraphics[width=\textwidth]{heat.png}
    \label{fig:heat}
  \end{subfigure}
  \begin{subfigure}[b]{0.4\textwidth}
    \caption{Metabolite Principal Components}
    \includegraphics[width=\textwidth]{pca.png}
    \label{fig:pca}
  \end{subfigure}
\end{figure}

The cross-validated classification error rate of the SPLS-DA model, for varying numbers of latent components, is reported in Figure \ref{fig:perf}. It appears that the accuracy of the model does not improve substantially after 2 components are introduced. This pattern appears to hold whether discriminant analysis is performed using maximum, centroids, or Mahalanobis distance. In our case, there is a slight case-control imbalance (41.2\% cases), but the patterns holds when adjusting for this imbalance with BER. For subsequent analyses, I restrict myself to 2 latent components.

Plotting the cross-validated ROC curve for predicting COPD (Figure \ref{fig:roc}, I observe an inflection point at which the sensitivity is 90\% and the specificity is 60\%. If I instead prioritize specificity, I observe a sensitivity of 40\% for a corresponding specificity of 90\%. The total area under the curve is 0.79. A more qualitative way of characterizing SPLS-DA model performance is to plot individuals according to their projection onto the first two latent variables. This plot is provided in Figure \ref{fig:indiv}, with individuals colored according to COPD status, overlaid onto the model's 95\% confidence ellipses. There is substantial overlap in the confidence ellipses for cases and controls, although cases on average are more represented in the bottom right quadrant of the plot.

The relative importance of each metabolite in the SPLS-DA model can be captured using the VIP (Variable Importance in the Projection) metric. A predictor with VIP greater than 1 is considered important in the model. I report the 15 metabolites with the highest VIP in Table \ref{tab:vip}. While I have not done a full literature review of previous COPD-metabolomic associations and have limited domain knowledge of COPD, several metabolites in this table grab my attention. VMA, the metabolite with the highest VIP, is a byproduct of the breakdown of epinephrine and norepinephrine, and as such is associated with physiological stress \cite{chapman29OrganicAcid2020}. While there may be some link between stress and COPD flare-ups \cite{COPDManagingStress}, this relationship is poorly understand and may not underly the association between VMA and COPD status in this cohort. Another metabolite in this table which stands out is thyroxine, an important hormone secreted by the thyroid. Thyroid dysfunction is known to be associated with COPD, although the mechanism and causal direction is unclear \cite{milkowska-dymanowskaThyroidGlandChronic2017}. A less ambiguous metabolite is hypotaurine, an intermediate in taurine metabolism, which has been associated with COPD status in multiple large studies \cite{godboleMetabolomeFeaturesCOPD2022}. Likewise, phosphocholine has been implicated with COPD in previous studies \cite{gillenwaterMetabolomicProfilingReveals2021}. Finally, it is worth noting the appearance of theophylline in this table. Theophylline is a common drug prescribed to treat the symptoms of COPD \cite{TheophyllineMedlinePlusDrug}. In this case, we know for certain the causal direction: use of theophylline is a response to COPD diagnosis. Although it is not a useful prognosticator of COPD status, it is encouraging to see our model identifying a metabolite which is unambiguously associated with COPD.

The GSEA analysis results are given in Figure \ref{fig:gsea}. We identify three enriched metabolomic sub-pathways for COPD status: pregnenolone steroids, androgenic steroids, and xanthine metabolism. I found that a number of metabolites categorized into the Xanthine metabolism sub-pathway were closely related to theophylline, indicating the result for this pathway may be spurious. There is some limited evidence implicating sex-related hormones, among which pregnenolone and androgenic steroids would be included, with COPD \cite{tamRoleFemaleHormones2011}.

\begin{figure}
  \centering
  \begin{subfigure}[b]{0.4\textwidth}
    \caption{PLS-DA Performance for Classifying COPD Status}
    \includegraphics[width=\textwidth]{plsdaPerf.png}
    \label{fig:perf}
  \end{subfigure}
  \begin{subfigure}[b]{0.45\textwidth}
    \caption{PLS-DA ROC Curve}
    \includegraphics[width=\textwidth]{roc.png}
    \centering
    \label{fig:roc}
  \end{subfigure}
\end{figure}

\input{../results/vip.tex}

\begin{figure}
  \begin{subfigure}[b]{0.45\textwidth}
  \caption{PLS-DA Individuals Plot}
  \includegraphics[width=\textwidth]{indiv.png}
  \label{fig:indiv}
  \end{subfigure}
\begin{subfigure}[b]{0.45\textwidth}
  \caption{GSEA Analysis: Enriched Subpathways}
  \includegraphics[width=\textwidth]{gsea.png}
  \label{fig:gsea}
  \end{subfigure}
  \end{figure}


\section*{Discussion}
My broad goal in this project was to identify metabolomic changes associated with COPD status. To do this, I analyzed metabolomic and phenotypic data of 636 subjects in the Plasma Metabolomic Signatures of COPD Study. Analysis of inter-metabolomic correlation revealed several blocks of highly-correlated metabolites, particularly among those related to lipid metabolism. Applying Gene Set Enrichment Analysis, I identified three metabolomic sub-pathways enriched for COPD status, motivating further inquiry into whether sex-related hormones levels differ for people with COPD. A Sparse Projection to Latent Structures Discriminant Analysis (SPLS-DA) model had cross-validated area under the curve of 0.79 for distinguishing between participants with and without COPD. Several important metabolites in the SPLS-DA model were validated by previous literature associations.

This project has a number of limitations. Participants were selected based on their history of smoking, limiting the applicability of these results to a general application. Lack of standardized metabolite nomenclature limiting my ability to validate individual metabolite associations. Finally, it is widely understood that age influences participants' metabolomic profiles as well as their COPD status, however I did not have access to this important confounder.

\addcontentsline{toc}{section}{Discussion}

\bibliographystyle{plain}
\bibliography{./references.bib}

\end{document}
